### PR TITLE
upgrades tar version from 6.0.2 to 6.1.11

### PR DESCRIPTION
**Pull request by Tenaria**
Upgrade tar version to 6.1.11 and run `npm install` to resolve security vulnerability. This is a minor bump and should not have breaking changes. `npm test` passes.

ticket: https://github.com/kubernetes-client/javascript/issues/769.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5688,8 +5688,7 @@
       "dev": true
     },
     "colors": {
-      "version": "1.4.0",
-      "resolved": "https://registry.npmjs.org/colors/-/colors-1.4.0.tgz",
+      "version": "https://registry.npmjs.org/colors/-/colors-1.4.0.tgz",
```
Comment (brendandburns) [inline package-lock.json:5691]: Any idea why the colors version is changing too?

File: package-lock.json [hunk 2]
```diff
@@ -8195,7 +8194,7 @@
       "integrity": "sha512-ZQZnjJPrt0rjp216gp6FQC1QC4ojcoKikhfOJ/51CqaJunVDilRLlIO5tCGWj1tzlYYT9eOGhJv7MF3t7rxSmw==",
       "dev": true,
       "requires": {
-        "colors": "^1.4.0",
```
Comment (brendandburns) [inline package-lock.json:8198]: This seems unrelated.

**Comment by brendandburns** [comment]
This looks fine, but there appears to be an unrelated change to the colors dependency.

**Comment by Tenaria** [comment]
@brendandburns I believe there was a typo introduced by this PR https://github.com/kubernetes-client/javascript/pull/767/files (see the first link on the PR description which recommends to downgrade colors to 1.4.0 but we pinned it to 1.40.0 which doesn't exist https://www.npmjs.com/package/colors). This is probably the root cause. I've pushed a fix.

File: package-lock.json (hunk 2 above)
Comment (Tenaria) [reply package-lock.json:8198]: (see comment in PR)

**Comment by Tenaria** [comment]
Thanks! @brendandburns 🙇

**Comment by Tenaria** [comment]
How often / how do you determine when to create a release btw? @brendandburns

Just wondering when I can expect this change to officially get released. Thanks!
